models/sudoswap/ethereum/sudoswap_ethereum_pool_creations.sql

```diff
@@ -0,0 +1,57 @@
+{{ config(
+        alias = 'pool_creations',
+        partition_by = ['pool_address'],
```

Comment:
i don't think it makes sense to partition by ```pool_address``` here, as it's unique and only one row per pool. spark doesn't love a ton of small files. we can remove partition altogether or look to use a different field, which would be commonly used in joins/filters for this table

Reply:
Ah yes, those shouldn't have stayed in.
Would be cool if we could partition on unique columns in the future with something like a mod 256 or so.